src/components/app/details/triggerView/ciMaterial.tsx

```diff
@@ -140,8 +140,17 @@ export class CIMaterial extends Component<CIMaterialProps, CIMaterialState> {
 
     renderCIModal(context) {
         let selectedMaterial = this.props.material.find((mat) => mat.isSelected)
+        let isMaterialActive = false
+        for(const material of this.props.material){
+            if(material.active){
+                isMaterialActive = true
+                break
+            }
+        }
         let canTrigger = this.props.material.reduce((isValid, mat) => {
-            isValid = isValid && !mat.isMaterialLoading && !!mat.history.find((history) => history.isSelected)
+            isValid =
+                (isValid && !mat.isMaterialLoading && !!mat.history.find((history) => history.isSelected)) ||
+                (mat.branchErrorMsg === 'Source not configured' && isMaterialActive)
```

Comment:
Move this error message to some const & use here and wherever we're assigning it `'Source not configured'`

Reply:
done